src/components/header/modals/ModalConnectWallet.vue

```diff
@@ -0,0 +1,154 @@
+      <div
+        v-for="(wallet, index) in nativeWallets"
+        :key="index"
+        class="box__row--wallet"
+        :wallet="wallet"
+        @click="setWalletModal(wallet.source)"
+      >
+        <div class="box--img">
+          <img width="40" :src="wallet.img" />
+        </div>
+        <div>{{ wallet.name }}</div>
+      </div>
+
+      <div class="title--account-type tw-mt-4">EVM Account</div>
+      <div
+        v-for="(wallet, index) in evmWallets"
+        :key="index"
+        class="box__row--wallet"
+        :wallet="wallet"
+        @click="setWalletModal(wallet.source)"
+      >
+        <div class="box--img">
+          <img width="40" :src="wallet.img" />
+        </div>
+        <div>{{ wallet.name }}</div>
+      </div>
+    </div>
+  </astar-simple-modal>
+</template>
+ <script lang="ts">
+import { defineComponent, watchEffect, ref } from 'vue';
+import { Wallet, supportNativeAccountWallets, supportEvmAccountWallets } from 'src/config/wallets';
+import { isMobileDevice } from 'src/hooks/helper/wallet';
+
+export default defineComponent({
+  props: {
+    isModalConnectWallet: {
+      type: Boolean,
+      required: true,
+    },
+    setCloseModal: {
+      type: Function,
+      required: true,
+    },
+    setWalletModal: {
+      type: Function,
+      required: true,
+    },
+    isEvmOnly: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
+  },
+  setup(props) {
+    const nativeWallets = ref<Wallet[]>(supportNativeAccountWallets);
+    watchEffect(() => {
+      nativeWallets.value = supportNativeAccountWallets
+        .map((it) => {
+          const { isSupportMobileApp, isSupportBrowserExtension } = it;
+          if (isMobileDevice) {
+            return isSupportMobileApp ? it : null;
+          } else {
+            return isSupportBrowserExtension ? it : null;
+          }
+        })
+        .filter((it) => it !== null) as Wallet[];
+    });
+
+    const evmWallets = ref<Wallet[]>(supportEvmAccountWallets);
+    watchEffect(() => {
+      evmWallets.value = supportEvmAccountWallets
+        .map((it) => {
+          const { isSupportMobileApp, isSupportBrowserExtension } = it;
+          if (isMobileDevice) {
+            return isSupportMobileApp ? it : null;
+          } else {
+            return isSupportBrowserExtension ? it : null;
+          }
+        })
+        .filter((it) => it !== null) as Wallet[];
+    });
+
+    return {
+      nativeWallets,
+      evmWallets,
+    };
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@import 'src/css/quasar.variables.scss';
+
+.wrapper--modal {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.title--account-type {
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 17px;
+  color: #2c3335;
+  text-align: left;
+  margin-left: 21px;
+}
+
+.box__row--wallet {
+  display: flex;
+  align-items: center;
+  background: #fff;
+  border-radius: 6px;
+  height: 3.5rem;
```

Comment:
May I know why did you use `rem` in this class?

Reply:
I know in general `rem` is better than `px` in terms of responsive design. but because of lack of time, I adjust small part of it.

Reply:
I see.
What do you think about making a 'rem function' that converts px to rem in case we need to use rem?
CC: @hoonsubin @bobo-k2 

```
styles/utils.scss
@function rem($px) {
  @return ($px / 16) * 1rem;
}

component/_button.scss
.my-button {
  font-size: rem(14); /* convert 14px to rem */
}

```

Reply:
Adding a util function seems like a reasonable solution.

Reply:
yeah sounds good!

Reply:
I've added rem function and converted px to rem in [this](https://github.com/AstarNetwork/astar-apps/pull/300) PR :)